app/controllers/bug_tickets_controller.rb

```diff
@@ -0,0 +1,42 @@
+class BugTicketsController < ApplicationController
+  before_action :find_ticket, only: [:edit, :update, :show, :destroy]
+  before_action :authenticated_only
+
+  def index
+    @user_bug_tickets = current_user.bug_tickets.page(params[:page]).reverse_order.decorate
+  end
+
+  def new
+    @bug_ticket = BugTicket.new
+  end
+
+  def create
+    current_user.bug_tickets.create!(bug_tickets_params)
+    redirect_to bug_tickets_path, notice: t('.ticket_created')
+  end
+
+  def update
+    @new_status = params['bug_ticket']['status']
+    respond_to do |format|
+      if @bug_ticket.update!(bug_tickets_params)
+        format.js
+      else
+        flash.now[:alert] = t('.not_updated')
+      end
+    end
+  end
+
+  def destroy
+    redirect_to bug_tickets_path, notice: t('.ticket_destroyed') if @bug_ticket.destroy!
```

Comment:
Wrong use bang (!) method here. So, we use bang method when we don't write any condition and want to see an error.

Example:

```ruby
@bug_ticket.destroy!

do_something
```

Without bang

```ruby
if @bug_ticket.destroy
...
else
...
end
```